src/resolve/resolved-gperf.gperf

```diff
@@ -33,3 +33,4 @@ Resolve.ResolveUnicastSingleLabel, config_parse_bool,                    0,
 Resolve.DNSStubListenerExtra,      config_parse_dns_stub_listener_extra, 0,                   offsetof(Manager, dns_extra_stub_listeners)
 Resolve.CacheFromLocalhost,        config_parse_bool,                    0,                   offsetof(Manager, cache_from_localhost)
 Resolve.StaleRetentionSec,         config_parse_sec,                     0,                   offsetof(Manager, stale_retention_usec)
+Resolve.RefuseRecordTypes,         config_parse_string,                  0,                   offsetof(Manager, refuse_record_types)
```

Comment:
please add your own config_parse_refuse_record_types() function, and parse the passed field via extract_first_word() + dns_type_from_string() piece-meal into a Set*.

Reply:
still not resolved

Reply:
@bluca I know it is not resolved as it's commented earlier on and pushing it is counter productive, unless you have some suggestions to add on to @poettering . So have some patience, i am trying to understand the code and i just started looking into systemd code in less than a week. When it's done, it's done.

Reply:
I mentioned it because the "good to merge" label was set, but it turns out it was a misclick: https://github.com/systemd/systemd/pull/34165#issuecomment-2320649919

Reply:
@poettering So i've added the value to `Set*` and how do i get this value in `resolved-dns-stub.c` ? It's not in some struct to store so i do not know how to get this in `resolved-dns-stub.c` from `conf-parser.c`

Is there some function called `set_` to get this?

`refused_record_types` is an int array in this case.
```
        _cleanup_set_free_free_ Set *refused_records;
        r = set_consume(refused_records, TAKE_PTR(refused_record_types));
        if (r < 0)
                return r;
```

Reply:
@bluca Oh. okay. Sorry for the misunderstanding!

Reply:
@poettering nevermind. forgot about the resolved manager. Now i can store it there and use it.